scopesim/effects/ter_curves_utils.py

```diff
@@ -50,9 +50,9 @@
 PATH_SVO_DATA = PATH_HERE.parent / "data" / "svo"
 
 
-def get_filter_effective_wavelength(filter_name):
+def get_filter_effective_wavelength(filter_name, cmds=None):
```

Comment:
Same as before, lets delete the currsys call

```suggestion
def get_filter_effective_wavelength(filter_name):
```